template/CHART_NAME/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
```

Comment:
Shouldn't we use `v2`?

Reply:
I added a comment in the Helm 2 EOL task in order to migrate the Chart.yaml (and requirements.yaml) at the same time we deprecate Helm 2 format.

Maintain this scaffolding up to date is something we're going to have to do. At this moment I used the current approach but taking into account it needs to be updated as part of the other task